src/dev_server.rs

```diff
@@ -49,11 +50,18 @@ use std::{
 /// }
 /// ```
 ///
-/// Add a `start` subcommand that will run `cargo xtask dist`, watching for changes in the workspace
-/// and serve the files in the default dist directory (`target/debug/dist` for non-release) at a
-/// given IP address.
+/// Add a `start` subcommand that will run `cargo xtask dist`, watching for
+/// changes in the workspace and serve the files in the default dist directory
+/// (`target/debug/dist` for non-release) at a given IP address.
 #[non_exhaustive]
 #[derive(Debug, clap::Parser)]
+#[clap(
+    about = "A simple HTTP server useful during development.",
+    long_about = "A simple HTTP server useful during development. \n\
+        It can watch the source code for changes and restart a provided \
+        command.
```

Comment:
The end user has no way to provide that command. It's a message for the help user.